### PR TITLE
Add a converter to convert from entity to the database object

### PR DESCRIPTION
**Pull request by arian487**
quick converter to go from entity object to database object

### CODE REVIEW

File: recidiviz/persistence/database/database_utils.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+from recidiviz.persistence.database import schema
+
+
+def convert_person_to_database(person):
+    """Converts the given person to a database person."""
+    person_db = schema.Person()
+    for k, v in vars(person).items():
+        if k == 'bookings':
+            person_db.bookings = [convert_booking_to_database(b) for b in
```
Comment (zacharylawrence) [inline recidiviz/persistence/database/database_utils.py]: Do you know if you can set the bookings on the ORM object to a `List` like this. You might need to use the `.extend()` method.

File: recidiviz/persistence/database/database_utils.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                                  person.bookings]
+        else:
+            setattr(person_db, k, v)
+    return person_db
+
+
+def convert_booking_to_database(booking):
```
Comment (zacharylawrence) [inline recidiviz/persistence/database/database_utils.py]: Does it make sense to make these private? Would we need to expose anything other than converting the top level `Person`?

File: recidiviz/persistence/database/database_utils.py (hunk 1 above)
Comment (arian487) [reply recidiviz/persistence/database/database_utils.py]: Discussed offline, it works

File: recidiviz/persistence/database/database_utils.py (hunk 2 above)
Comment (arian487) [reply recidiviz/persistence/database/database_utils.py]: Done.

**Pull request merged**